wasp/boards/pinetime/watch.py

```diff
@@ -0,0 +1,136 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
```

Comment:
this file should not be committed

Reply:
confused as to how that file even changed, ye i can make it 3 seperate pull requests, Im gonna close this one and make 3 new ones, sorry for the bad commits